index.js

```diff
@@ -37,6 +38,10 @@ module.exports = robot => {
         if (!(await PullRequestBodyChecker.isBodyValid(body, config, context))) {
           badBody = true
         }
+      } else if (eventSrc === 'issue') {
+        if (!(await IssueBodyChecker.isBodyValid(body, config, context))) {
```

Comment:
What do you think about check `config.checkIssueTemplate` here?

Reply:
This was me following the pattern that `PullRequestBodyChecker` used. I can move it out, just wanting to confirm I should ignore that.

Reply:
This is very minor, so I'll defer to you on whether matching the existing code or being slightly more efficient is better. Let me know which you decide @shiftkey and happy to merge this whenever.

Reply:
How about we leave this as-is, and I'll follow up with a PR to lift both this `config.checkIssueTemplate` and `config.checkPullRequestTemplate` out of their relevant components and into here, so they remain consistent?

Reply:
@hiimbex friendly bump :wave: